src/registry.js

```diff
@@ -24,6 +24,12 @@ class SimpleLogger {
     arguments[0] = 'ERROR: ' + arguments[0];
     console.log.apply(console, arguments);
   }
+
+  trace() {}
+
+  isLevelEnabled(level) {
+    return level !== 'trace';
```

Comment:
How would one enable trace level logging? Would they just need to pass in their own implementation? It seems like it could be handy to have some basic knobs on the SimpleLogger to just tweak the level that is enabled.

Should the level specific methods check what level is enabled? Otherwise it would be easy to have an inconsistency where the level is not enabled, but when calling `log.{level}("message")` it still writes out the message.

Reply:
You're not supposed to use the default logger for production. This is just for tests / development without introducing an extra dependency. Users of the package will provide their own logging implementation, in which case the configuration is done through their specific mechanisms. I'll add a `setLevel` for consistency but I don't want to make the impression that this is a general logging framework and encourage its use.